mls-rs/src/group/mod.rs

```diff
@@ -265,8 +280,7 @@ where
     private_tree: TreeKemPrivate,
     key_schedule: KeySchedule,
     #[cfg(feature = "by_ref_proposal")]
-    pending_updates:
-        crate::map::SmallMap<HpkePublicKey, (HpkeSecretKey, Option<SignatureSecretKey>)>, // Hash of leaf node hpke public key to secret key
+    pending_updates: SmallMap<HpkePublicKey, PendingUpdate>, // Hash of leaf node hpke public key to secret key
```

Comment:
@mulmarta I _think_ that we persist this data to disk as part of the group state itself? If so making this change would break loading group states from before this version. We can probably just try to deserialize the new format followed by the old format somehow.